tests/core/snap-set-core-config/task.yaml

```diff
@@ -141,5 +141,11 @@ execute: |
     echo "setting the timezone works"
     cat /etc/timezone > current-timezone
     snap set system system.timezone=Europe/Malta
-    MATCH "Europe/Malta" /etc/timezone
+    MATCH "Europe/Malta" < /etc/timezone
     test "$(readlink -f /etc/localtime)" = "/usr/share/zoneinfo/Europe/Malta"
+    echo "but the timezone setting is not stored in the state"
+    snap get system -d | not grep "timezone"
```

Comment:
IMHO this is surprising, I'd think hijacked key behaves like any other key and the fact it's not stored in the state is an implementation detail and it should be reported when complete document was requested?